runtime/doc/msgpack_rpc.txt

```diff
@@ -413,5 +417,9 @@ of update.
 ["popupmenu_hide"]
 	The popupmenu is hidden.
 
+["tabline", curtab, tabs]
```

Comment:
An event is something that happens. Let's name this something more granular, e.g. "tabline_update". That follows the pattern of the existing "bg_update" and similar events.